common/app/conf/switches.scala

```diff
@@ -702,6 +702,15 @@ object Switches {
     exposeClientSide = true
   )
 
+  val ImgixAllImagesSwitch = Switch(
+    "Feature",
+    "imgix-all-images",
+    "If this switch is on, then all images will be served via the third party image resizing service",
+    safeState = On,
```

Comment:
`On` by default so it is enabled in dev environments - this is fine for PROD too as the other switch for Imgix is the panic switch.